### PR TITLE
Update command help

### PR DESCRIPTION
**Pull request by twcsamar**

### CODE REVIEW

**Comment by stof** [comment]
this would make it inaccurate for anyone using the Symfony 2.x structure. Please switch to use ``%command.full_name%`` instead, letting Symfony figure out what should be used. This will make it compatible with any project structure.

**Comment by Ener-Getick** [comment]
@twcsamar can you do that ?

**Comment by OskarStark** [comment]
ping @twcsamar

File: Command/ActivateUserCommand.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ protected function configure()
             ->setHelp(<<<EOT
 The <info>fos:user:activate</info> command activates a user (so they will be able to log in):
 
-  <info>php app/console fos:user:activate matthieu</info>
+  <info>php %command.full_name%/console fos:user:activate matthieu</info>
```
Comment (OskarStark) [inline Command/ActivateUserCommand.php]:
```diff
- <info>php %command.full_name%/console fos:user:activate matthieu</info>
+ <info>php %command.full_name% matthieu</info>
```

**Comment by OskarStark** [comment]
Thank you @twcsamar 👍 

This PR looks good and is RTM

**Comment by XWB** [comment]
Thanks guys.

**Pull request merged**